### ISSUE TITLE
Use multiple tags to select services

### ISSUE THREAD

**Issue by thy485280869** [mbobakov/grpc-consul-resolver#34]
Does this item only support one label for filtering? I want to use multiple tags to select services (for example: tags [] string), just like the official ServiceMultipleTags() method, but this project does not seem to support it.
![85C`)JUO5)Q{B 7L5XJ30HN](https://user-images.githubusercontent.com/72010101/200948891-10b61e62-4633-4c3e-963e-14c7234336dc.png)

**Comment by mbobakov** [comment]
Good point. I will mark it as good first issue. If nobody come i will look into it myself. 
ETA is end of April.
